packages/webapp/src/components/Finances/PureExpenseDetail/index.jsx

```diff
@@ -0,0 +1,177 @@
+/*
+ *  Copyright 2023 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useForm, Controller } from 'react-hook-form';
+import Form from '../../Form';
+import PageTitle from '../../PageTitle/v2';
+import Input, { getInputErrors } from '../../Form/Input';
+import Button from '../../Form/Button';
+import PropTypes from 'prop-types';
+import { IconLink } from '../../Typography';
+import DeleteBox from '../../Task/TaskReadOnly/DeleteBox';
+import { ReactComponent as TrashIcon } from '../../../assets/images/document/trash.svg';
+import { hookFormMaxCharsValidation } from '../../Form/hookformValidationUtils';
+import { useCurrencySymbol } from '../../../containers/hooks/useCurrencySymbol';
+import ReactSelect from '../../Form/ReactSelect';
+import { getDateFromDateTimeString } from '../../../util/moment';
+import { NOTE, VALUE, DATE, TYPE } from '../AddExpense/constants';
+
+const PureExpenseDetail = ({
+  pageTitle,
+  handleGoBack,
+  onSubmit = () => {},
+  onRetire = () => {},
+  view,
+  buttonText,
+  inputMaxChars = 100,
+  expense,
+  expenseTypeReactSelectOptions,
+}) => {
+  const { t } = useTranslation();
+  const [isDeleting, setIsDeleting] = useState(false);
+  const {
+    handleSubmit,
+    register,
+    control,
+    formState: { errors, isValid, isDirty },
+  } = useForm({
+    mode: 'onChange',
+    defaultValues: {
+      [NOTE]: expense.note,
+      [DATE]: getDateFromDateTimeString(expense.expense_date),
+      [TYPE]: expenseTypeReactSelectOptions.find(
+        (option) => option.value === expense.expense_type_id,
+      ),
+      [VALUE]: expense.value,
+    },
+  });
+  const readonly = view === 'read-only' || false;
```

Comment:
`|| false` could be removed 😉

Reply:
I think I'm going to remove it from the source component where that code comes from, too.

Reply:
Oops deleted a comment, but... I'm thinking I should remove it from the source component where that code comes from too?